src/bundle/EventListener/RequestListener.php

```diff
@@ -42,7 +42,10 @@ public static function getSubscribedEvents(): array
     {
         return [
             // 10001 is to ensure that REST requests are tagged before CorsListener is called
-            KernelEvents::REQUEST => ['onKernelRequest', 10001],
+            KernelEvents::REQUEST => [
+                ['onKernelRequest', 10001],
```

Comment:
Wasn't there a cap of 255 for this value?

Reply:
Why would there be a limit? :sweat_smile: 

It can be any number, including maximum and minimum integer values. It's up to developer to pick right value.

(also note I'm not changing the priority).

Reply:
> Why would there be a limit? 😅
> 
> It can be any number, including maximum and minimum integer values. It's up to developer to pick right value.
> 
> (also note I'm not changing the priority).

So this was the part of doc that was the source of my confusion

> The priorities of the internal Symfony listeners usually range from -256 to 256 but your own listeners can use any positive or negative integer.

All good then :sweat_smile: